src/main/kotlin/ar/edu/unahur/obj2/semillasAlViento/Parcela.kt

```diff
@@ -2,44 +2,29 @@ package ar.edu.unahur.obj2.semillasAlViento
 
 class Parcela(val ancho: Int, val largo: Int, val horasSolPorDia: Int) {
   val plantas = mutableListOf<Planta>()
-  var cantidadPlantas = 0
+
```

Comment:
En vez de borrarlo podrías haber cambiado que `cantidadPlantas` en vez de ser una `var` se calcule.
```suggestion
  fun cantidadPlantas() = plantas.size
```